communication/messages/messages.go

```diff
@@ -31,20 +31,39 @@ type StructureRequest struct {
 }
 
 type StructureResponse struct {
-	Type         string                  `json:"type"`
-	Blockchain   blockchain.Blockchain   `json:"blockchain"`
-	Transactions blockchain.Transactions `json:"transactions"`
+	Type       string                `json:"type"`
+	Blockchain blockchain.Blockchain `json:"blockchain"`
 }
```

Comment:
Skoro 1 node ma działać tylko to nie będę implementować integracji transakcji pomiędzy node'ami